apps/src/sites/studio/pages/levels/_curriculum_reference.js

```diff
@@ -1,6 +1,16 @@
-window.curriculumReference = {
-  onload: onload
-};
+import $ from 'jquery';
+import { registerGetResult } from '@cdo/apps/code-studio/levels/codeStudioLevels';
+import { postMilestoneForPageLoad, onContinue } from '@cdo/apps/code-studio/levels/postOnLoad';
+
+$(document).ready(() => {
```

Comment:
I always find myself a little confused when I see the shorthand, and am apparently not alone in this considering their docs have the warning: 
` If you are writing code that people who aren't experienced with jQuery may see, it's best to use the long form.`